masterlist.yaml

```diff
@@ -17632,12 +17910,15 @@ plugins:
       - 'URN+CoT Patch.esp'
       - 'Atlas Map Markers.esp'
       - 'The Horkerpocalypse.esp'
-    tag:
-      - C.Location
```

Comment:
I followed the order [here](https://loot.github.io/docs/0.9.2/LOOT%20Metadata%20Syntax.html) that seems to be used by the majority of the masterlist.